core/vibes/soul/sections/footer/index.tsx

```diff
@@ -0,0 +1,372 @@
+import { clsx } from 'clsx';
+import Image from 'next/image';
```

Comment:
```suggestion
import { BcImage as Image } from '~/components/bc-image';
```

Reply:
We need to use BcImage here for the logo image.

Reply:
We will leave as is since we will change this after #1710 changes gets added to the `soul/main` branch.